libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -112,6 +111,7 @@ class MapboxTripNotification constructor(
     override fun getNotificationId(): Int = NOTIFICATION_ID
 
     override fun updateNotification(routeProgress: RouteProgress) {
+        buildRemoteViews()
```

Comment:
👀 https://github.com/mapbox/mapbox-navigation-android/pull/1474 same change in the legacy didn't solve it 😥

Reply:
Ah I see. Well considering that it is responsible for many crashes, we can at least mitigate